libzim/wrapper.pyx

```diff
@@ -370,6 +370,18 @@ cdef class FilePy:
         """Get the filename of the File object"""
         return self._filename
 
+    def has_article(self, url):
```

Comment:
I'm not sure we need this method.

It may let think the check is less expensive than getting directly the article, but in fact this is the same.

As the user code will probably be :
```python
if zimfile.has_article("foo"):
    article = zimfile.get_article("foo")
else:
    print("Not found")
```

We will end to call twice `get_article`.

If there is no method :
```python
try:
    article = zimfile.get_article("foo")
except NotFound:
    print("Not found")
```

It will call the `get_article` only once and it follow the [Easier to ask for forgiveness than permission](https://docs.python.org/3/glossary.html#term-eafp ) principle.

(The EAFP principle doesn't forbid a `has_article` method, but as the implementation of `has_article` is a call to `get_article` I think we should not have it)

Reply:
yeah you mentioned that would be revamped so was hoping we'd get something similar then. Will go away for now.